docs/Lexicon.md

```diff
@@ -102,6 +102,11 @@
 
 ### R
 
+* RAUW
+  Replace All Uses With
+  A term commonly used in LLVM referring to a method that replaces all the uses
```

Comment:
check "view" for this change ;)

Reply:
wtf?  what's different between this and all the other defs below it?

Reply:
Omg.  Is it seriously the two spaces after "Replace All Uses With"?  Here I thought I liked Markdown.